src/components/ActivitiesListFilter/ActivitiesListFilter.js

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 import Input from '../Input/Input';
 
 const ActivitiesListFilter = () => (
-  <div>
+  <>
```

Comment:
there is only `<Input />` returned from this function, so `Fragment` is redundant.